src/app/lib/config.js

```diff
@@ -52,6 +52,8 @@
 
     types_yts: ['All', '720p', '1080p', '2160p', '3D'],
 
+    ratings_yts: ['All', '1+', '2+', '3+', '4+', '5+', '6+', '7+', '8+', '9+'],
```

Comment:
This isnt necessary, just add them as 'All', '1', '2'.. etc here as they were and change
`<li><a href="#" data-value="<%= rating %>"><%= i18n.__(rating.capitalizeEach()) %></a></li>`
in filter-bar.tpl to
`<li><a href="#" data-value="<%= rating %>"><%= i18n.__(rating.capitalizeEach()) %> + '+'</a></li>`

Reply:
don't know, basically you want to keep your strings as they are displayed. Just to be able to search them in the project. Especially with these particular ratings that are numbers.
if in future someone needs to lookup where the plus comes from, they'll spend a lot of time searching for it.

So I'd rather keep them as they are, but of course it's up to you.